### PR TITLE
test1

### PR DESCRIPTION
**Pull request by gxfang1985**

### CODE REVIEW

File: source/test.php [hunk 1]
```diff
@@ -3,11 +3,12 @@
 
 function myTest($x)
 {
-  echo $y + $x['name'];
+  echo $y + $x['name'] + $z;
```
Comment (gxfang1985) [inline source/test.php:6]: /wtd Remove wrong variable

File: source/test.php [hunk 2]
```diff
@@ -3,11 +3,12 @@
...
 }
 
 myTest($a);
 
 $query = "select * from" .$_GET['input'];
 $result = mysqli_query($query);
 
+echo 'test1';
```
Comment (gxfang1985) [inline source/test.php:14]: /wtd delete this line

File: source/test.php [hunk 3]
```diff
@@ -11,4 +11,6 @@ function myTest($x)
 $query = "select * from" .$_GET['input'];
 $result = mysqli_query($query);
 
-echo 'test1';
\ No newline at end of file
+echo 'test1';
+
+echo 'test1-add
```
Comment (gxfang1985) [inline source/test.php:16]: /wtd   please delete this line

File: source/test.php [hunk 4]
```diff
@@ -3,11 +3,14 @@
...
 }
 
 myTest($a);
 
 $query = "select * from" .$_GET['input'];
 $result = mysqli_query($query);
 
+echo 'test1';
+
+echo 'test1-add
```
Comment (cybozulijia) [inline source/test.php:16]: /wtd delete this line

Comment (gxfang1985) [inline source/test.php:16]: /WTD delete these lines

**Pull request merged**